source/_integrations/lock.mqtt.markdown

```diff
@@ -69,6 +69,14 @@ availability_topic:
   description: The MQTT topic subscribed to receive availability (online/offline) updates. Must not be used together with `availability`.
   required: false
   type: string
+code_format:
+  description: A regular expression to validate a supplied code when it is set during the service call to `open`, `lock` or `unlock` the MQTT lock.
+  required: false
+  type: string
+command_template:
+  description: Defines a [template](/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to generate the payload to send to `command_topic`. The lock command template accepts the parameters `value` and `code`. The `value` parameter will contain the configured value for either `payload_open`, `payload_lock` or `payload_unlock`. The `code` parameter is only available if it is set during the service call to `open`, `lock` or `unlock` the MQTT lock.
```

Comment:
> The `code` parameter is only available if it is set during the service call to `open`, `lock` or `unlock` the MQTT lock.

If `code_format` is set, that's not a problem since the service call will fail. If `code_format` is NOT set, maybe we should include a template example which shows how to handle that, for example by setting code to JSON `null`?